js/source/geojson_source.js

```diff
@@ -42,10 +46,18 @@ function GeoJSONSource(options) {
 
     if (options.maxzoom !== undefined) this.maxzoom = options.maxzoom;
 
-    this.geojsonVtOptions = { maxZoom: this.maxzoom };
+    this.geojsonVtOptions = {maxZoom: this.maxzoom};
     if (options.buffer !== undefined) this.geojsonVtOptions.buffer = options.buffer;
     if (options.tolerance !== undefined) this.geojsonVtOptions.tolerance = options.tolerance;
 
+    this.cluster = options.cluster || false;
+    this.superclusterOptions = {
+        maxZoom: Math.max(options.clusterMaxZoom, this.maxzoom - 1) || (this.maxzoom - 1),
+        extent: 4096,
+        radius: options.clusterRadius || 400,
```

Comment:
Why is the default here `400` but the [default in supercluster](https://github.com/mapbox/supercluster/blob/master/index.js#L20) `40`?